W-13180007 The Result Message after a scatter gather (#2503)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/scatter-gather-concept.adoc

WARNING: Configure your Scatter-Gather component to have at least two routes; otherwise, your Mule application throws an exception and does not start.

== Result

> The Result Message after a scatter gather of each route is stored in a result map in the form

Reviewer: ```suggestion
After all routes complete, the component outputs results in the following format:
```

> `{"0":messageFromRoute0,"1":messageFromRoute1,...}`

Reviewer 2: ```suggestion
`{0: messageFromRoute0, 1: messageFromRoute1, ...}`
```
I think it would be more correct to not put semicolons.

> To extract all resulting payloads to an array you can use

Reviewer: ```suggestion
To extract all resulting payloads to an array, use a DataWeave script, such as the following one:
```

> 

Reviewer: TODO: let's get verification of this script on the output. Do we want header directives. Also, can we do this in Logger or should we use Transform Message?

Author: As this is just a sniped which may be used for set-payload or set-variable or in a more complex dataweave I think header (%dw) and output type can be omitted here.
